crates/matrix-sdk/src/widget/machine/mod.rs

```diff
@@ -121,6 +135,9 @@ pub(crate) struct WidgetMachine {
     /// Outstanding requests sent to the matrix driver (mapped by uuid).
     pending_matrix_driver_requests: PendingRequests<MatrixDriverRequestMeta>,
 
+    /// Outstanding state updates waiting to be sent to the widget.
+    pending_state_updates: Option<InitialStateUpdate>,
```

Comment:
Please explain in the doc comment a few more things: why is this needed? when is it set? when it's unset, will it stay unset forever, or can it be reset later?